site/en/blog/prerender-pages/index.md

```diff
@@ -179,7 +194,25 @@ Multiple speculation rules can also be added to the same page, and they append t
 </script>
 ```
 
-#### Detecting of Speculation Rules API support
+### Speculation rules restrictions and future enhancements
+
+By default prerender is restricted to same-origin pages, opened within the same tab, but we are working to reduce those restrictions.
```

Comment:
Maybe this should be "same-site" now, since per the next aside, that's the current boundary?

Reply:
I guess that's true. But we're not planning to change the "default" in that sense, so then the last half of the sentence feels a bit off.

Reply:
Ah gotcha.

WDYT about this?:

> At present speculation rules are restricted to pages opened within the same tab, but we are working to reduce that restrictions. By default prerender is restricted to same-origin pages.

Reply:
Updated to that suggestion in https://github.com/GoogleChrome/developer.chrome.com/pull/6004/commits/6af7f216ec227ff6dae496c6b46e6396f00ffc7f

Reply:
LGTM!